### ISSUE TITLE
set params when training the flow model

### ISSUE THREAD

**Issue by arunos728** [mit-han-lab/temporal-shift-module#12]
I'm trying to reproduce the two-stream results of TSM on Something v1, but the performance of my flow model is far below. (segment based sampling method)

I understand the 10 channels stacked optical flow (TV-L1) / learning rate 5 times in the first conv layer.

Is there any difference between RGB and Flow model in setting params?
(e.x, epochs, learning rate...)

**Comment by tonylins** [comment]
Hi, we basically used the same training hyper-parameters for flow models. How much accuracy can you get using optical flow? Note that optical flow model should give a lower performance compared to RGB input.

**Comment by arunos728** [comment]
I just stopped the training process before 50 epochs cuz my performance is too low.
It was lower than 30% at 40epochs. (ResNet-18)
Could you tell me the performance of TSM Flow model on Something v1? (8 segments, single inference, ResNet-50)

**Comment by tonylins** [comment]
The accuracy of TSM Flow model on Something V1 is about 42%.

**Comment by arunos728** [comment]
Thanks for the answer.
I just modified the dense sampling to the uniform sampling, and achieves the similar performance as you said.

**Comment by spireheat** [comment]
> Thanks for the answer.
> I just modified the dense sampling to the uniform sampling, and achieves the similar performance as you said.

Hi, I just trained the TSM-ResNet50 model on SomethingV1 with 8-segments optical flow input only, but the performance is around 37%, far below the 42%, and the two-stream is around 48%. Can you provide the link to download the optical flow for SomethingV1 dataset that you use?

**Comment by Fritskee** [comment]
@arunos728 How did you prepare the optical-flow data? Was it as suggested in the [Temporal Segment Network (TSN)](https://github.com/yjxiong/temporal-segment-networks#extract-frames-and-optical-flow-images) repository, which requires  the [denseflow](https://github.com/shuangshuangguo/dense_flow) package?

**Comment by jianghaojun** [comment]
@boheumd How did you extract flow data for sthv1?
